statsmodels/distributions/transforms_copula.py

```diff
@@ -0,0 +1,268 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Wed Jan 27 14:33:40 2021
+
+Author: Josef Perktold
+License: BSD-3
+
+"""
+
```

Comment:
Would also make more sense to be in distributions/copula/transforms.py rather than using file names to sort of create namespace.